### ISSUE TITLE
[BUG] Navbar content not visible (4.0.0)

### ISSUE THREAD

**Issue by federico-ntr** [italia/design-react-kit#785]
**Describe the bug**
In 4.0.0 I cannot see the links inside the navbar, either in the stories and using the component from the package.

**To Reproduce**
1. Go to [docs](https://italia.github.io/design-react-kit/?path=/docs/componenti-header--header-nav#header-nav)
2. See that navbar is just a blue rectangle with no links even though they seem to be in the source

**Expected behavior**
Just like in 3.3.3 the navbar should show the links

**Screenshots**
![image](https://user-images.githubusercontent.com/35468654/142854234-e325d838-f267-4bad-930d-e3f5ca7a278b.png)
It happens also on header complete:
![image](https://user-images.githubusercontent.com/35468654/142854389-0b8d4bc8-f083-4e39-9cf6-00a466c03205.png)

**Other Required Information (please complete the following information):**
 - OS: Windows 10
 - Browser Chrome 96

**Comment by federico-ntr** [comment]
I found out the problem the issue is due to the [last commit](https://github.com/italia/design-react-kit/commit/5fc8e01f9e459cc01af7fd0aa02b8d0bad06f8bd) on Collapse component (line 59):
```
transform: isOpen ? 'translateX(0)' : 'translateX(-100%)'
```
`translateX(-100%)` is applied to hide the "drawer" menu on mobile. It's a smart trick but the style is applied on the component also without a mobile resolution and makes the links disappear (or better, it makes the links go outside the container).

We cannot simply add a media query since the style is in a style tag. I don't think it's difficult to solve this issue but I don't have a great understanding of the styling system. I'll give it a try anyway.

**Comment by dej611** [comment]
Is this something that happens on doc pages only, or it can be reproduced/can cause problems in a full page as well?

Because if it’s a doc issue only we could address it locally as with other doc stories as well (there’s a storybook-fixes.css file somewhere for that)

**Comment by federico-ntr** [comment]
No, it happens also in our company's application. I would suggest to revert that commit for now and to find another way to animate the menu opening. 

I actually don't understand why the animation doesn't work without that commit since I found the CSS properties applied to the elements are the same as the BI ones. Does it have to do with how they are applied? I tried to mess around a bit with the elements but I didn't manage to find out hot it works
